packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -19,18 +19,17 @@ import {StrategyMath} from "./base/StrategyMath.sol";
 
 /**
  * Migration Error Codes:
- * M1: Crab V2 Address already set
- * M2: Migration already happened
- * M3: Migration has not yet happened
- * M4: msg.sender is not Euler Mainnet Contract
- * M5: msg. sender cannot send ETH
- * M6: Can't withdraw more than you own
- * M7: Not enough ETH to repay the loan
+ * M1: Migration already happened
+ * M2: Migration has not yet happened
+ * M3: msg.sender is not Euler Mainnet Contract
+ * M4: msg.sender cannot send ETH
+ * M5: Can't withdraw more than you own
+ * M6: Not enough ETH to repay the loan
+ * M7: Invalid crabV2 address
```

Comment:
Mixed with M8? 
        require(_ethToBorrow > 0 && _withdrawMaxEthToPay > 0, "M7");

M8: _ethToBorrow or _withdrawMaxEthToPay can't be 0

Reply:
I believe the require statement is `require(_ethToBorrow > 0 && _withdrawMaxEthToPay > 0, "M8")` and tests also use M8?

Reply:
ah you're right - was looking at main

Reply:
What line do you see ```require(_ethToBorrow > 0 && _withdrawMaxEthToPay > 0, "M7");```

I see that being M8 in the code and in the error legend.